NDTensors/src/lib/Sectors/src/category_product.jl

```diff
@@ -0,0 +1,107 @@
+
+struct CategoryProduct{Labels} <: AbstractCategory
+  labels::Labels
+  global _CategoryProduct(l) = new{typeof(l)}(l)
+end
+
+CategoryProduct(c::CategoryProduct) = _CategoryProduct(labels(c))
+
+labels(s::CategoryProduct) = s.labels
+
+Base.isempty(S::CategoryProduct) = isempty(labels(S))
+Base.length(S::CategoryProduct) = length(labels(S))
+Base.getindex(S::CategoryProduct, args...) = getindex(labels(S), args...)
+
+function fusion_rule(s1::CategoryProduct, s2::CategoryProduct)
+  return [CategoryProduct(l) for l in labels_fusion_rule(labels(s1), labels(s2))]
+end
+
+Base.:(==)(A::CategoryProduct, B::CategoryProduct) = labels_equal(labels(A), labels(B))
+
+const Sector = CategoryProduct
```

Comment:
I would vote for defining a `sector` function and using that instead of `Sector`.